src/server_manager/electron_app/index.ts

```diff
@@ -26,16 +27,19 @@ const app = electron.app;
 const ipcMain = electron.ipcMain;
 const shell = electron.shell;
 
+// Run before referencing environment variables.
+dotenv.config({path: path.join(__dirname, '.env')});
+
 const debugMode = process.env.OUTLINE_DEBUG === 'true';
 
 const IMAGES_BASENAME =
     `${path.join(__dirname.replace('app.asar', 'app.asar.unpacked'), 'server_manager', 'web_app')}`;
 
-const sentryDsn =
-    process.env.SENTRY_DSN || 'https://533e56d1b2d64314bd6092a574e6d0f1@sentry.io/215496';
+const sentryDsn = process.env.SENTRY_DSN;
 
 sentry.init({
-  dsn: sentryDsn,
+  // Error reporting is a no-op when `sentryDsn` is undefined.
+  dsn: sentryDsn || '',
```

Comment:
Should we just not call init if sentryDsn is empty?

Reply:
I prefer this, since it is possible to "submit feedback". Not initializing the SDK may throw an error when attempting to submit feedback.